### ISSUE TITLE
initialization order

### ISSUE THREAD

**Issue by dalehenrich** [dalehenrich/Rowan#40]
Currently the initialization order is unspecified:
```smalltalk
runInitializers

	"run the class initialization methods as needed"

	(addedMethods copy
		addAll: extendedMethods;
		addAll: methodsNeedingRecompile;
		yourself) do: [ :methodPatch | methodPatch runInitializer ]
```
RwPackageProjectSpecification>>packageNamesInLoadOrder should be renamed to RwPackageProjectSpecification>>packageNamesInInitializationOrder (load order is immaterial because of atomic loading - at least at the moment) and the the initializers should be run in package initialization order (first) and class hierarchy order (secondary) ...

**Comment by dalehenrich** [comment]
I think there is code in Monticello and Cypress for ordering initialization methods ... probably ought to adapt them for Rowan

**Comment by dalehenrich** [comment]
fixed with pr #112 merge
